main.tf

```diff
@@ -142,6 +142,7 @@ resource "google_service_account_iam_binding" "workload_identity_binding" {
 }
 
 resource "null_resource" "cred_config_json" {
+  count = var.skip_local_exec == true ? 0 : 1
```

Comment:
Do we need to put this condition in output also

Reply:
Lets continue to output the command so they can run it outside of Terraform.